packages/data-stores/src/automated-transfer-eligibility/selectors.ts

```diff
@@ -1,97 +0,0 @@
-import { createRegistrySelector } from '@wordpress/data';
-import { STORE_KEY, statusMapping } from './constants';
-import {
-	State,
-	TransferEligibilityWarning,
-	TransferEligibilityError,
-	TransferEligibility,
-} from './types';
-
-export const getAutomatedTransferEligibility = (
-	state: State,
-	siteId: number | null
-): TransferEligibility | null => {
-	if ( ! siteId ) {
-		return null;
-	}
-	return state[ siteId ];
-};
-
-export const getEligibilityHolds = createRegistrySelector(
-	( select ) =>
-		( state: State, siteId: number | null ): TransferEligibilityError[] | null => {
-			const transferEligibility = select( STORE_KEY ).getAutomatedTransferEligibility(
-				siteId
-			) as TransferEligibility | null;
-
-			if ( ! transferEligibility ) {
-				return null;
-			}
-
-			const { errors } = transferEligibility;
-
-			if ( ! errors ) {
-				return null;
-			}
-
-			return errors.map( ( { code } ) => {
-				return statusMapping[ code ] ?? '';
-			} );
```

Comment:
At a quick look, it also looks like this selector was not returning a `TransferEligibilityError[]`, since the `TransferEligibilityError` type has a `code` and a `message` properties, while it seems that the return statement here returns `string[]`

Could this mean that, prior to this PR, the call to `getEligibilityHolds` was buggy? How can we test that?